docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -708,7 +708,7 @@ Result:
 
 - Alias: mismatches
 
-## byteJaccardIndex
+## jaccardIndex
```

Comment:
This expects we will quickly add this function back.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/55126